### PR TITLE
feat : product page added

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my product page

### CODE REVIEW

File: Styles/product.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+.filter-price span{
+    padding: 1rem 2rem;
+    font-weight: bolder;
+    font: 1.2em sans-serif;
+}
+
+.range{
+    display: flex;
+    justify-content: space-around;
```
Comment (PuneetP16) [inline Styles/product.css]: Suggestion: uneven spacing could be fixed.

File: index.html [hunk 2]
```diff
@@ -34,6 +34,12 @@
         >
```
Comment (PuneetP16) [inline index.html:34]: image

On hover, it's scaling too much, Could be fixed.

Comment (PuneetP16) [inline index.html:34]: UX suggestion: filters checkbox and text don't look aligned. Maybe could be vertically aligned.
image

**Review by PuneetP16 — APPROVED** [review]
Code looks good! suggested changes in UX part. Looks good to merge after necessary changes.

File: Styles/product.css (hunk 1 above)
Comment (yashodaAgarwal) [reply Styles/product.css]: fixed

**Review by iHarryD — COMMENTED** [review]
Would recommend using uppercase letters for variable names. It would help in distinguishing variables from classes.

File: index.html (hunk 2 above)
Comment (yashodaAgarwal) [reply index.html:34]: Thankyou hadnot checked on it.Fixed now

**Pull request merged**